contracts/src/libs/MerkleTree.sol

```diff
@@ -141,21 +151,25 @@ library MerkleTree {
     /// @param self The tree data structure.
     /// @return treeDepth The depth of the tree.
```

Comment:
What is the definition of tree depth here? Assuming that this term is equivalent to tree height...